### PR TITLE
Add sceptre instance with ssh files

### PR DESCRIPTION
**Pull request by tthyer**
See if sceptre will create the instance. The toil ami, as yet unencrypted, has been copied into scicomp.

### CODE REVIEW

File: config/prod/rna-seq-reprocessing-instance-v001.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  # The resource owner
+  OwnerEmail: "tess.thyer@sagebase.org"
+
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "toil"
+  AMIId: "ami-032e0d8fe5cfa8c7f"  # toil-ubuntu ami
+  RoleStackName: "rna-seq-reprocessing-instance-role-v001"
+
+  RsaKey: !ssm /toil-cluster-infra/SshKey
+  PublicRsaKey: !ssm /toil-cluster-infra/PublicSshKey
```
Comment (tthyer) [inline config/prod/rna-seq-reprocessing-instance-v001.yaml]: @zaro0508 I make these secure string params in sandbox, but will need your assistance to get them into scicomp.

File: templates/toil-infra-essentials.yaml [hunk 2]
```diff
@@ -44,6 +45,7 @@ Resources:
                 - !ImportValue us-east-1-bootstrap-TravisUserArn
                 - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
                 - !ImportValue us-east-1-rna-seq-reprocessing-instance-role-v001-ToilClusterRoleArn
+                - "arn:aws:iam::563295687221:role/accounts-AWSIAMAdminRole-1B5HUQPC19H69"
```
Comment (zaro0508) [inline templates/toil-infra-essentials.yaml]: same here with condition

File: templates/toil-infra-essentials.yaml [hunk 3]
```diff
@@ -22,6 +22,7 @@ Resources:
                     - ':root'
                 - !ImportValue us-east-1-bootstrap-TravisUserArn
                 - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
+                - "arn:aws:iam::563295687221:role/accounts-AWSIAMAdminRole-1B5HUQPC19H69"
```
Comment (zaro0508) [inline templates/toil-infra-essentials.yaml]: this is ONLY for sandbox, it's different for scicomp.  You'll need to add [conditional logic](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html) to set this for sandbox or scicomp depending on which ENV you run it for.  Condition is used in  toil-instance.yaml

File: config/dev/config.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+project_code: sage-bionetworks
```
Comment (zaro0508) [inline config/dev/config.yaml]: sceptre configs support inheritance which means this config will inherit any configs from config/config.yaml so you don't need to repeat everything in this config.  You only need to append additional params specific to dev env.  You'll also want to add a config/prod/config.yaml file as well.

File: templates/toil-infra-essentials.yaml (hunk 3 above)
Comment (zaro0508) [reply templates/toil-infra-essentials.yaml]: the equivalent role in scicomp is "arn:aws:iam::055273631518:role/accounts-AWSIAMAdminRole-JL0BF315YDU0"

File: templates/toil-instance.yaml [hunk 5]
```diff
@@ -58,6 +58,12 @@ Parameters:
     Default: 8
     MinValue: 8
     MaxValue: 1000
+  RsaKey:
```
Comment (zaro0508) [inline templates/toil-instance.yaml]: where's the `EncryptVolume` parameter?

**Comment by tthyer** [comment]
Some of the feedback was not addressed before merge. It is handled in PR #35.

**Pull request merged**